src/main/kotlin/com/rroperations/models/Train.kt

```diff
@@ -1,9 +1,13 @@
 package com.rroperations.models
 
+import java.util.UUID
+
 data class Train(
         val nameOfCar: String,
         val destination: String,
-        val receiver: String) {
+        val receiver: String,
+        var classificationTrack: String? = null,
+        var trainId: UUID? = null) {
 
     override fun toString(): String {
```

Comment:
Maybe we will need to edit the toString method to include the classificationTrack when is not null, something like 
`return if (classificationTrack == null) "Train(nameOfCar='$nameOfCar', destination='$destination', receiver='$receiver')" else "Train(classificationTrack='$classificationTrack', nameOfCar='$nameOfCar', destination='$destination', receiver='$receiver')"`

Reply:
I checked the default data class toString() method, it prints all attributes regardless of their value. So the null ones get printed also. Do you think it is necessary to hide the null ones?